lib/src/yaml_node_wrapper.dart

```diff
@@ -41,6 +41,10 @@ class YamlMapWrapper extends MapBase
     return value;
   }
 
+  @override
+  Map<String, dynamic> asJsonType() =>
+      Map<String, dynamic>.from(_dartMap.cast<String, dynamic>());
```

Comment:
Using both `Map.from` and `.cast` seems redundant... We should also be using inference here.

How about `Map.from(_dartMap);`